app/portainer/components/forms/git-form/git-form-auto-update-fieldset/git-form-auto-update-fieldset.html

```diff
@@ -1,4 +1,10 @@
-<ng-form name="autoUpdateForm">
+<ng-form name="autoUpdateForm form-group">
+  <div class="small vertical-center">
+    <pr-icon icon="'info'" mode="'primary'"></pr-icon>
+    <span class="text-muted">
+      When enabled, at each polling interval or webhook invocation, if the git repo differs from what was stored locally on last git pull, the changes are deployed.</span
```

Comment:
"on last git pull" or "on the last git pull" ?

Reply:
PO question

Reply:
updated